examples/connect-app-teleop-react/README.md

```diff
@@ -0,0 +1,80 @@
+# React App and Tele-operation Example
+
+This example project allows you to browse your machines and stream a video from a Viam Rover and control its movements with your keyboard or mouse.
+
+## Authentication Setup
+
+There are two authentication options for this app:
+1. API key id and secret baked into the frontend
+1. OAuth2 via FusionAuth
+
+### API Key
+
+Go to your organization settings and either create or copy the information for an API key listed under "API Keys". The environment variables you'll need in the next step from this are:
+
+```ini
+VITE_APP_ORG_ID=yourorgid
+VITE_APP_API_KEY_ID=21d425b4-0aed-49da-82c9-1e9bda895863
+VITE_APP_API_KEY_SECRET=somesecret
+```
+
+### FusionAuth
+
+For FusionAuth, you'll need to have set up the [Viam CLI](https://docs.viam.com/cli/). Using the CLI, set up an auth app:
+
+```shell
+BASE_DEV_URI="http://localhost:5173"
+BASE_PROD_URI="http://localhost:9000"
+viam auth-app register --application-name my-app --org-id "yourorgid" --redirect-uris "$BASE_DEV_URI,$BASE_PROD_URI,$BASE_DEV_URI/app/callback,$BASE_PROD_URI/app/callback" --origin-uris "$BASE_DEV_URI,$BASE_PROD_URI"
+```
+
+Note: If you want to use Firefox, you'll need a tool like [ngrok](https://ngrok.com/) since Firefox does not allow WebRTC on localhost hosted pages. You will also need to update the URI list in the command above in addition to using the `VITE_BASE_URI` environment variable (or set in .env) referring to your ngrok (or other proxy) endpoint.
```

Comment:
slight change for clarity
```suggestion
Note: If you want to use Firefox, you'll need a tool like [ngrok](https://ngrok.com/) since Firefox does not allow WebRTC on localhost hosted pages.
You will also need to update the URI list in the command above in addition to using the `VITE_BASE_URI` environment variable (or set it in .env) to use your ngrok (or other proxy) endpoint.
```